packages/ui/src/components/sha-badge.tsx

```diff
@@ -1,4 +1,4 @@
-import { Text } from '@harnessio/canary'
+import { Text } from '@/components'
 
 interface RootProps {
   children: React.ReactNode
```

Comment:
Looks like `React` isn't imported in this file. Can we import `{ ReactNode } from 'react'?